modules/media/jsx/mediaIndex.js

```diff
@@ -101,7 +156,7 @@ class MediaIndex extends React.Component {
             Data={this.state.Data.Data}
```

Comment:
As of PR #3955, hiddenHeaders need to be passed as `props` to `StaticDataTable`.

Reply:
Done